youtube_dl/extractor/acast.py

```diff
@@ -22,7 +23,8 @@ class ACastIE(InfoExtractor):
             'id': '57de3baa-4bb0-487e-9418-2692c1277a34',
             'ext': 'mp3',
             'title': '"Where Are You?": Taipei 101, Taiwan',
-            'timestamp': 1196172000000,
+            'timestamp': 1196172000,
+            'upload_date': '20071127',
```

Comment:
Not sure about this. This is because of https://github.com/rg3/youtube-dl/blob/master/youtube_dl/YoutubeDL.py#L1287